features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/assets/default/asset.jag

```diff
@@ -47,9 +47,23 @@ require('/modules/publisher.js').exec(function(ctx) {
     var filterManager = publisher.filterManager;
 
     mod = jagg.module("documentation");
-
     var typeList = apiProvider.getEnabledAssetTypeList();
 
+    function getOwnerList() {
+        var businessOwnerList = apiProvider.getBusinessOwnerList();
+        var ownersList = JSON.parse(stringify(businessOwnerList));
```

Comment:
Seems that `APIProvider.getBusinessOwnerList()` returns a `List<BusinessOwner>` (see [line 280](https://github.com/wso2/carbon-appmgt/pull/278/files#diff-01d9b3af51a76ea6a0a5b61090a1c279R280)). My suggestion is, instead converting the Java `List` to a JS array access the it in a Java manner. You can use `List.size()` method to figure-out the length of the list and `List.get(index)` to get a value.